examples/feature_flags/src/custom_s3_store_provider.py

```diff
@@ -22,10 +22,8 @@ def __init__(self, bucket_name: str, object_key: str):
 
     def _get_s3_object(self) -> Dict[str, Any]:
         # Retrieve the object content
-        parameters = {"Bucket": self.bucket_name, "Key": self.object_key}
-
         try:
-            response = self.client.get_object(**parameters)
+            response = self.client.get_object(Bucket=self.bucket_name, Key=self.object_key)
```

Comment:
It works best this way with the introduced mypy_boto3 implicit type annotations. Otherwise mypy would not be able to check the types of the parameters. I'm happy you agree it looks better too!